pkg/metrics/metrics.go

```diff
@@ -56,3 +58,59 @@ func WithMetricsMultiAddress(multiAddrs []MultiAddrs) error {
 	_, err = meter.RegisterCallback(callback, multiAddressesGauge)
 	return err
 }
+
+// BlockHeight represents the information for the block height 1.
+type BlockHeight struct {
+	ServiceName string  // ServiceName Name of the service associated with the multiaddress.
+	BlockHeight string  // Namespace where the service is deployed.
+	Value       float64 // Value to be observed for the multiaddress.
+}
+
+// WithMetricsBlockHeight creates a callback function to observe metrics for block_height_1.
+// consensus-node:26657/block?height=1
+func WithMetricsBlockHeight(blockHeight, earliestBlockTime, serviceName, namespace string) error {
+	log.Info("registering metric: ", blockHeight)
+	// Create a Float64ObservableGauge named "block_height_1" with a description for the metric.
+	blockHeightGauge, err := meter.Float64ObservableGauge(
+		"block_height_1",
+		metric.WithDescription("Torch - BlockHeight"),
+	)
+	if err != nil {
+		log.Fatalf(err.Error())
+		return err
+	}
+	callback := func(ctx context.Context, observer metric.Observer) error {
+		// Define the callback function that will be called periodically to observe metrics.
+		// Create labels with attributes for each block_height_1.
+		labels := metric.WithAttributes(
+			attribute.String("service_name", serviceName),
+			attribute.String("block_height_1", blockHeight),
+			attribute.String("earliest_block_time", earliestBlockTime),
+			attribute.Int("days_running", CalculateDaysDifference(earliestBlockTime)),
```

Comment:
that would be great, but for now, I think is not possible due to the Grafana transformation doesn't have this feature yet to add calculations to attributes
I tried it before and I couldn't find a way to do it...

Reply:
Interesting. What is the reason for having the "days running" as an attribute and not as a metric value? Like a timestamp.

(I don't think we should change your code, as it works. I want to understand different methods)

Reply:
I guess blockchain hash can't be a metric value, as it's a string, right?

Reply:
> Interesting. What is the reason for having the "days running" as an attribute and not as a metric value? Like a timestamp.
> 
> (I don't think we should change your code, as it works. I want to understand different methods)

do you mean to set the value of the metric to the `days running`?, I think it is more clear to use an attribute, because the metric is called `earliest_block_time` which contains more data, another option could be just to create a new metric called `days_running`(or whatever), and set the value to it 🤔 I don't discard this option, we can split this info in a different metrics

wdyt?

the change is tiny, I'll only need a good name for the metric 😌

Reply:
I would say no, we are using the metric type: [Float64ObservableGauge](https://github.com/jrmanes/torch/blob/main/pkg/metrics/metrics.go#L74), and the blockchain hash is a string... that's why we are setting the value [1](https://github.com/jrmanes/torch/blob/main/pkg/metrics/metrics.go#L93) to the metric (*also to make it like activated*)

Reply:
Thanks for the explanation!